### PR TITLE
Small cleanups

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by bgeuken** [comment]
This one depends on #52

Created a new PR since those two changes don't belong together. Hope that's okay.

File: lib/cli.rb [hunk 1]
```diff
@@ -265,8 +261,8 @@ def organization_members
 
     trello = TrelloWrapper.new(@@settings)
 
-    members = trello.organization(options["org-name"]).members
-    members.sort! { |a,b| a.username <=> b.username }
+    members = trello.organization(options["org-name"]).
+                sort! { |a,b| a.username <=> b.username }
```
Comment (bgeuken) [inline lib/cli.rb]: Oh, right. I'll change it

[the remark this answers is not in the document]
Comment (Ana06) [reply lib/cli.rb]: @bgeuken ping!

**Comment by bgeuken** [comment]
@Ana06 rebased

**Pull request merged**